packagecontrol.py

```diff
@@ -98,6 +98,7 @@ def packages_json_generate():
                            Package.date,
                            Package.version,
                            Package.download_url,
+                           Package.download_count,

```

Comment:
the `packages.json` does not need this information. The keypirinha plugin has no use for this value, therefore it can be omitted for efficiency reasons.